static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-us-message-copy-experiment.js

```diff
@@ -0,0 +1,71 @@
+define([
+    'bean',
+    'reqwest',
+    'fastdom',
+    'qwery',
+    'common/utils/$',
+    'common/utils/config',
+    'common/modules/commercial/commercial-features',
+    'common/utils/mediator'
+], function (
+    bean,
+    reqwest,
+    fastdom,
+    qwery,
+    $,
+    config,
+    commercialFeatures,
+    mediator
+) {
+    return function () {
+        this.id = 'MembershipEngagementUsMessageCopyExperiment';
+        this.start = '2016-10-27';
+        this.expiry = '2016-11-15';
+        this.author = 'Justin Pinner';
+        this.description = 'Test alternate short messages on engagement banner';
+        this.audience = 0.5;
+        this.audienceOffset = 0;
```

Comment:
Note that with this [`audienceOffset`](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md#writing-a-test), this test will coincide with all users participating in [`membership-engagement-warp-factor-one`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-warp-factor-one.js) and some users participating in [`membership-engagement-message-copy-experiment`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/tests/membership-engagement-message-copy-experiment.js)